### PR TITLE
Feat: cli as dev deps

### PR DESCRIPTION
**Pull request by Grubba27**
relates to #12135 

--- 

## Maintainers Checklist:

- [ ] release new meteor version for the changed deps

### CODE REVIEW

File: npm-packages/meteor-installer/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,761 @@
+{
+  "name": "meteor",
+  "version": "2.8.0",
```
Comment (henriquealbert) [inline npm-packages/meteor-installer/package-lock.json:3]: is this version correct?

Comment (Grubba27) [reply npm-packages/meteor-installer/package-lock.json]: I should not have committed this file. For some reason, there is not a json.lock file in this package

**Comment by Grubba27** [comment]
Hey @zodern a friendly ping to see if you by any change had the time to review this one

**Comment by Grubba27** [comment]
For now we decided to discuss more about this one. Will reopen when we have a more clear idea in how to proceed

**Pull request closed without merging**